app/templates/briefs/edit_brief_response_question.html

```diff
@@ -0,0 +1,33 @@
+{% extends "briefs/_base_edit_question_page.html" %}
+
+{% block breadcrumb %}
+
+  {%
+    with items = [
+      {
+        "link": "/",
+        "label": "Digital Marketplace"
+      },
+      {
+        "link": "/{}/opportunities".format(brief.frameworkSlug),
+        "label": "Supplier opportunities"
+      },
+      {
+        "link": "/{}/opportunities/{}".format(brief.frameworkSlug, brief.id),
+        "label": brief.title
+      },
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block save_button %}
+  {%
+    with
+    label="Continue" if not is_last_page else "Submit application",
```

Comment:
Maybe `"Submit application" if is_last_page else "Continue"` to get rid of the negative